src/Components/LifecycleChart/lifecycleChart.tsx

```diff
@@ -0,0 +1,131 @@
+      ],      
+    ];
+
+
+
+    const formatDate = (date: Date, isTime: boolean) => {
+      const dateString = date?.toLocaleDateString("en-US", { year: 'numeric' });
+      const timeString = date?.toLocaleTimeString("en-US", { hour12: false });
+      return isTime ? `${dateString} ${timeString}` : dateString;
+    };
+
+    const getChart = (alert: any[], index: number) => {
+      const data: any[] = [];
+
+      alert?.map((datum: { packageType: string; }) => {
+        data.push({
+          ...datum,
+          x: alerts.length - index,
+          fill: datum.packageType === "Retired"
+            ? "var(--pf-v5-global--danger-color--100)"
+            : datum.packageType === "Support ends within 6 months"
+              ? "var(--pf-v5-global--warning-color--100)"
+              : datum.packageType === "Not installed"
+                ? "var(--pf-v5-global--palette--blue-200)"
+                : datum.packageType === "Supported"
+                  ? "var(--pf-v5-global--success-color--100)"
+                    : datum.packageType === "Upcoming release"
+                      ? "var(--pf-v5-global--palette--blue-100)"
+                      : "var(--pf-v5-global--default-color--300)",
+        });
+      });
+
+      if (data?.length === 0) {
+        return null;
+      }
+      return (
+        <ChartBar
+          data={data}
+          key={index}
+          style={{
+            data: {
+              fill: ({ datum }) => datum.fill,
+              stroke: ({ datum }) => datum.fill,
+            }
+          }}
+        />
+      );
+    };
+
+    return (
+      <Card>
+        <div style={{ height: "400px", width: "450px" }}>
+          <Chart
+            ariaDesc="Support timelines of packages and RHEL versions"
+            ariaTitle="Lifecycle bar chart"
+            containerComponent={
+              <ChartVoronoiContainer
+                labelComponent={
+                  <ChartTooltip constrainToVisibleArea labelComponent={<ChartLabel dx={-65} textAnchor="start" />} />
+                }
+                labels={({ datum }) => `Package: ${datum.x}\npackageType: ${datum.packageType}\nStart: ${formatDate(new Date(datum.y0), true)}\nEnd: ${formatDate(new Date(datum.y), true)}`}
+              />
+            }
+            domainPadding={{ x: [20, 20], y: [40, 40] }}
+            legendData={[
+              { name: "Supported", symbol: { fill: "var(--pf-v5-global--success-color--100)" } },
+              { name: "Support ends within 6 months", symbol: { fill: "var(--pf-v5-global--warning-color--100)" } },
+              { name: "Retired", symbol: { fill: "var(--pf-v5-global--danger-color--100)" } },
+              { name: "Not installed", symbol: { fill: "var(--pf-v5-global--palette--blue-200)" } },
+              { name: "Upcoming release", symbol: { fill: "var(--pf-v5-global--palette--blue-100)" } }
+            ]}
+            legendPosition="bottom-left"
+            height={400}
```

Comment:
This is fine for now as a first pass, but we'll want to make this full-width later. I have a Jira ticket for this, so no need to address right now.